src/roman_datamodels/datamodels.py

```diff
@@ -179,6 +180,16 @@ def __setattr__(self, attr, value):
     def __getattr__(self, attr):
         return getattr(self._instance, attr)
 
+    def __setitem__(self, key, value):
+        if key.startswith('_'):
+            raise ValueError(
+                'May not specify attributes/keys that start with _')
+        if hasattr(self._instance, key):
+            print('XXXXXX')
```

Comment:
Sir, I caught this print statement nosing around the methods.  Do you want me to send it to the brig?

Reply:
Damn, I'm so imperfect. No, I'll delete it.